### PR TITLE
Updates the README.md

### LINKED ISSUE #11: Update README.md to reflect build, run example and documentation changes

### PR DESCRIPTION
**Pull request by jasinai**
It reflects the new build process for qabel-desktop.
Closes #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,27 +10,16 @@ Desktop Frontend of Qabel
 
 0. Make sure you have a working [git client](http://git-scm.com/) installed
 
-0. clone and build your submodules
+0. install everything from `requirements` and do `building source` from the [qabel README.md](https://github.com/Qabel/qabel/blob/master/README.md)
 
-   have a look at https://github.com/Qabel/qabel-helloworld-module/ for an example
-  
-0. clone the source
+0. build the jars from inside the qabel-core folder
```
Comment (thechauffeur) [inline README.md]: I get

```
qabel-core]$ ./gradlew jar

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':compileJava'.
> Could not determine the dependencies of task ':compileJava'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 3.586 secs
```

Are you sure it is from inside the `qabel-core` folder?

File: README.md [hunk 2]
```diff
@@ -10,27 +10,16 @@ Desktop Frontend of Qabel
 
 0. Make sure you have a working [git client](http://git-scm.com/) installed
```
Comment (thechauffeur) [inline README.md]: I would leave this step out.

Comment (thechauffeur) [inline README.md]: It is mentioned in the referenced README below.

File: README.md [hunk 3]
```diff
@@ -10,27 +10,16 @@ Desktop Frontend of Qabel
...
-   git submodule update
-   git submodule foreach git submodule init
-   git submodule foreach git submodule update
    ./gradlew jar
    ```
 0. run (example with helloworld-module)
 
    ```
+   cd qabel-desktop
    java -Djava.library.path=qabel-core -cp "qabel-helloworld-module/build/libs/qabel-helloworld-module-0.1.jar:qabel-desktop/build/libs/qabel-desktop-0.1.jar" de.qabel.desktop.QblMain -module qabel-hellowor-module/build/libs/qabel-helloworld-module-0.1.jar:de.qabel.helloworld.QblHelloWorldModule
```
Comment (thechauffeur) [inline README.md]: This only works from the `qabel` repository not `qabel-desktop`.

Comment (thechauffeur) [inline README.md]: When run from the `qabel` repository I get
```
qabel]$ java -Djava.library.path=qabel-core -cp "qabel-helloworld-module/build/libs/qabel-helloworld-module-0.1.jar:qabel-desktop/build/libs/qabel-desktop-0.1.jar" de.qabel.desktop.QblMain -module qabel-hellowor-module/build/libs/qabel-helloworld-module-0.1.jar:de.qabel.helloworld.QblHelloWorldModule
2015-07-06 17:23:29.819 [main] INFO  de.qabel.core.config.SQLitePersistence - Cannot select SALT from CONFIG database!
java.sql.SQLException: ResultSet closed
	at org.sqlite.core.CoreResultSet.checkOpen(CoreResultSet.java:66) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.findColumn(JDBC3ResultSet.java:39) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.getBytes(JDBC3ResultSet.java:270) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getConfigValue(SQLitePersistence.java:170) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.getSalt(SQLitePersistence.java:58) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.Persistence.<init>(Persistence.java:49) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.<init>(SQLitePersistence.java:32) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:125) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.077 [main] INFO  de.qabel.core.config.SQLitePersistence - Cannot select MASTERKEYNONCE from CONFIG database!
java.sql.SQLException: ResultSet closed
	at org.sqlite.core.CoreResultSet.checkOpen(CoreResultSet.java:66) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.findColumn(JDBC3ResultSet.java:39) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.getBytes(JDBC3ResultSet.java:270) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getConfigValue(SQLitePersistence.java:170) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.getMasterKey(SQLitePersistence.java:74) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.Persistence.<init>(Persistence.java:49) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.<init>(SQLitePersistence.java:32) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:125) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.148 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.Contact)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:70) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.150 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.Account)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:73) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.151 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.DropServer)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:77) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.153 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.Identity)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:81) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.154 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.LocaleModuleSettings)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:85) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.156 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.LocalSettings)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:89) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.158 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.StorageServer)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:93) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.159 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.StorageVolume)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:97) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.160 [main] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entities!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.SyncedModuleSettings)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Statement.executeQuery(JDBC3Statement.java:81) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntities(SQLitePersistence.java:355) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.loadFromPersistence(ResourceActor.java:101) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.<init>(ResourceActor.java:66) [qabel-core-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.<init>(QblMain.java:127) [qabel-desktop-0.1.jar:0.1]
	at de.qabel.desktop.QblMain.main(QblMain.java:36) [qabel-desktop-0.1.jar:0.1]
2015-07-06 17:23:30.232 [ConfigActor] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entity!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.DropServer)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.CorePreparedStatement.<init>(CorePreparedStatement.java:40) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3PreparedStatement.<init>(JDBC3PreparedStatement.java:29) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc4.JDBC4PreparedStatement.<init>(JDBC4PreparedStatement.java:18) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:47) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:251) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:223) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntity(SQLitePersistence.java:335) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.updateOrPersistEntity(SQLitePersistence.java:294) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.react(ResourceActor.java:520) [qabel-core-0.1.jar:0.1]
	at de.qabel.ackack.Actor$3.run(Actor.java:108) [ackack-0.1.jar:?]
	at de.qabel.ackack.Actor.run(Actor.java:143) [ackack-0.1.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]
2015-07-06 17:23:30.268 [ConfigActor] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entity!
java.sql.SQLException: ResultSet closed
	at org.sqlite.core.CoreResultSet.checkOpen(CoreResultSet.java:66) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.findColumn(JDBC3ResultSet.java:39) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.getBytes(JDBC3ResultSet.java:270) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntity(SQLitePersistence.java:338) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.updateOrPersistEntity(SQLitePersistence.java:294) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.react(ResourceActor.java:520) [qabel-core-0.1.jar:0.1]
	at de.qabel.ackack.Actor$3.run(Actor.java:108) [ackack-0.1.jar:?]
	at de.qabel.ackack.Actor.run(Actor.java:143) [ackack-0.1.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]
2015-07-06 17:23:30.294 [ConfigActor] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entity!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.Identity)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.CorePreparedStatement.<init>(CorePreparedStatement.java:40) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3PreparedStatement.<init>(JDBC3PreparedStatement.java:29) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc4.JDBC4PreparedStatement.<init>(JDBC4PreparedStatement.java:18) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:47) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:251) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:223) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntity(SQLitePersistence.java:335) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.updateOrPersistEntity(SQLitePersistence.java:294) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.react(ResourceActor.java:531) [qabel-core-0.1.jar:0.1]
	at de.qabel.ackack.Actor$3.run(Actor.java:108) [ackack-0.1.jar:?]
	at de.qabel.ackack.Actor.run(Actor.java:143) [ackack-0.1.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]
2015-07-06 17:23:30.324 [ConfigActor] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entity!
java.sql.SQLException: ResultSet closed
	at org.sqlite.core.CoreResultSet.checkOpen(CoreResultSet.java:66) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.findColumn(JDBC3ResultSet.java:39) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.getBytes(JDBC3ResultSet.java:270) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntity(SQLitePersistence.java:338) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.updateOrPersistEntity(SQLitePersistence.java:294) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.react(ResourceActor.java:531) [qabel-core-0.1.jar:0.1]
	at de.qabel.ackack.Actor$3.run(Actor.java:108) [ackack-0.1.jar:?]
	at de.qabel.ackack.Actor.run(Actor.java:143) [ackack-0.1.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]
2015-07-06 17:23:30.348 [ConfigActor] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entity!
java.sql.SQLException: [SQLITE_ERROR] SQL error or missing database (no such table: de.qabel.core.config.Contact)
	at org.sqlite.core.DB.newSQLException(DB.java:890) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.newSQLException(DB.java:901) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.throwex(DB.java:868) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.NativeDB.prepare(Native Method) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.DB.prepare(DB.java:211) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.core.CorePreparedStatement.<init>(CorePreparedStatement.java:40) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3PreparedStatement.<init>(JDBC3PreparedStatement.java:29) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc4.JDBC4PreparedStatement.<init>(JDBC4PreparedStatement.java:18) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:47) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:251) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:223) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntity(SQLitePersistence.java:335) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.updateOrPersistEntity(SQLitePersistence.java:294) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.react(ResourceActor.java:498) [qabel-core-0.1.jar:0.1]
	at de.qabel.ackack.Actor$3.run(Actor.java:108) [ackack-0.1.jar:?]
	at de.qabel.ackack.Actor.run(Actor.java:143) [ackack-0.1.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]
2015-07-06 17:23:30.386 [ConfigActor] ERROR de.qabel.core.config.SQLitePersistence - Cannot get entity!
java.sql.SQLException: ResultSet closed
	at org.sqlite.core.CoreResultSet.checkOpen(CoreResultSet.java:66) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.findColumn(JDBC3ResultSet.java:39) ~[sqlite-jdbc-3.8.7.jar:?]
	at org.sqlite.jdbc3.JDBC3ResultSet.getBytes(JDBC3ResultSet.java:270) ~[sqlite-jdbc-3.8.7.jar:?]
	at de.qabel.core.config.SQLitePersistence.getEntity(SQLitePersistence.java:338) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.SQLitePersistence.updateOrPersistEntity(SQLitePersistence.java:294) [qabel-core-0.1.jar:0.1]
	at de.qabel.core.config.ResourceActor.react(ResourceActor.java:498) [qabel-core-0.1.jar:0.1]
	at de.qabel.ackack.Actor$3.run(Actor.java:108) [ackack-0.1.jar:?]
	at de.qabel.ackack.Actor.run(Actor.java:143) [ackack-0.1.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_79]
```

Also I guess the servers are not running if someone follows this README. They need to get started manually prior to issuing this command (I would reference the corresponding section from the `qabel` README).

Comment (jasinai) [inline README.md]: I can't do anything about the SQLExceptions. But yes, I forgot to start the servers.

**Pull request merged**